gateway.go

```diff
@@ -90,7 +92,14 @@ func main() {
 		log.Fatalf("Failed to create gateway configuration from seed: %s", err)
 	}
 
-	gateway := ohttp.NewDefaultGateway(config)
+	var gateway ohttp.Gateway
+	requestLabel := os.Getenv(customRequestEncodingType)
+	responseLabel := os.Getenv(customResponseEncodingType)
+	if requestLabel == "" || responseLabel == "" || requestLabel == responseLabel {
```

Comment:
Can I ask if it will it work correctly if requestLabel == responseLabel == "protobuf" ?